addons/jaeger/jaeger.yaml

```diff
@@ -8,13 +8,13 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: jaeger
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "2.18.3-4"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.18.3-5"
     appversion.kubeaddons.mesosphere.io/jaeger-operator: "1.21.2"
     appversion.kubeaddons.mesosphere.io/jaeger: "1.21.0"
     stage.kubeaddons.mesosphere.io/jaeger: Experimental
     endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
-    docs.kubeaddons.mesosphere.io/jaeger: "https://www.jaegertracing.io/docs/1.21/"
-    values.chart.helm.kubeaddons.mesosphere.io/jaeger: "https://raw.githubusercontent.com/jaegertracing/helm-charts/aa3b160/charts/jaeger-operator/values.yaml"
+    docs.kubeaddons.mesosphere.io/jaeger: "https://www.jaegertracing.io/docs/1.20/"
```

Comment:
How is this field set?

@cgawron1 Do you know who now owns the dispatch instance/script that creates this PRs?

Reply:
This script is run from a cron via Dispatch.   This is the script that is executed 
https://github.com/mesosphere/dispatch-tasks/blob/master/bump_charts/bump_charts.py

bump_charts is owned by DevProd

Reply:
I find that bump_charts script very hard to understand and read.

Reply:
looks pretty straight-forward to me. /shrug

We can still throw @cprovencher under the bus for it though. He's not here to defend himself. ;)

Reply:
@joejulian - I take back. yeah, it is not hard and not even a hard regex. Just was not sure why.

Reply:
@cgawron1 I took a quick look at the script and did not find anything. I believe that this should be investigate as the version is wrong.

Who should own it? I can take a deeper look into it, but @drewvanstone would need to approve.

Reply:
The script grabs the version to bump from the apiversion. In this case this version was overwritten by hand by grabbing it from website. This situation is an edge-case and given the human requirement to manually review these bump it is recommended to leave as is. Thanks @joejulian for the insights.